### DISCUSSION TITLE
Where do I find the Discover Weekly playlist?

### DISCUSSION THREAD

**Discussion by Gnorv** [KRTirtho/spotube#1266] [Q&A] [answered]
In the [changelog under V3.0.0](https://github.com/KRTirtho/spotube/releases/tag/v3.0.0) the Discover Weekly playlist of Spotify is mentioned as something that is available.
After doing the login process with the flatpak and/or android version I cannot find where it is. Does anyone have a suggestion?

**Comment by KRTirtho** [comment, marked as answer]
Does it appear on the official Spotify app? For new users, it'll get generated after using Spotify at least a week

**Reply by Gnorv** [reply]
You are right, also in the official Spotify app I do not see the Discover Weekly playlist. I have used Spotify a few years ago and started to reuse the account with Spotube now. I guess after some time of inactivity Spotify stops generating a Discover Weekly playlist for you. Does Spotube share play-history with Spotify, so that Spotify can use it to generate a Discover Weekly playlist?

**Reply by KRTirtho** [reply]
> Does Spotube share play-history with Spotify, so that Spotify can use it to generate a Discover Weekly playlist?

No, unfortunately. The playback history is entirely locally stored
